docusaurus/docs/form/select/components/resource-select.md

```diff
@@ -125,6 +125,10 @@ When true, the network request is not made until the dropdown has been focused.
 
 When true, if the `resource` only returns one result the first time it is called, the value is defaulted to the single result. Note: if `waitUntilFocused` is `true`, this prop is ignored.
 
+#### `defaultToFirstOption?: boolean`
+
+When true, if the `resource` only returns at least one result the first time it is called, the value is defaulted to the first result. Note: if `waitUntilFocused` is `true`, this prop is ignored.
```

Comment:
This is probably from copy/pasting, but should it be "if the resource returns at least one result" instead?